README.md

```diff
@@ -16,13 +16,24 @@ dependencies:
 
 - `eslint`
 
-- `husky` (allows you specify git hook scripts in your `package.json`)
+- `husky` (allows you specify git hook scripts inside `.husky` -
```

Comment:
Can you change to `.husky/` (which trailing slash) to make it clear that they need to files to that directory?